Habitica/src/com/magicmicky/habitrpgwrapper/lib/models/tasks/Task.java

```diff
@@ -112,8 +119,8 @@ public void setId(String id) {
     /**
      * @return the notes
      */
-    public String getNotes() {
-        return notes;
+    public CharSequence getNotes() {
+        return markdownParser.parseMarkdown(notes);
```

Comment:
I can move the Tasks stuff into their ViewAdapters, I must still change the type of `text` and `notes` from `String` to `CharSequence`. Can I do this?